### DISCUSSION TITLE
Saving the brightness settings

### DISCUSSION THREAD

**Discussion by n3ptun3s** [SynthstromAudible/DelugeFirmware#229] [Ideas]
It would be great, if brightness would also appear in the normal settings/community feature page and be saved after adjusting.

**Comment by sapphire-arches** [comment]
If you're up for writing a bit of code, this shouldn't be too hard to add. The relevant steps would be:
- Add a new entry to https://github.com/SynthstromAudible/DelugeFirmware/blob/cd0a7024ff62be381188c624b6101309cf5d8c6a/src/deluge/model/settings/runtime_feature_settings.h
- Create a menu_item subclass similar to https://github.com/SynthstromAudible/DelugeFirmware/blob/cd0a7024ff62be381188c624b6101309cf5d8c6a/src/deluge/gui/menu_item/osc/retrigger_phase.h, but pointing at [this variable](https://github.com/SynthstromAudible/DelugeFirmware/blob/cd0a7024ff62be381188c624b6101309cf5d8c6a/src/deluge/hid/led/pad_leds.cpp#L695C5-L695C16), probably via a new API `getRefreshTime()` [around here](https://github.com/SynthstromAudible/DelugeFirmware/blob/cd0a7024ff62be381188c624b6101309cf5d8c6a/src/deluge/hid/led/pad_leds.h#L75)

**Comment by saynotohugs** [comment]
+ 1 to saving brightness settings :)

**Comment by joxihan** [comment]
I think if people are interested I can make the change

**Comment by lobit12** [comment]
+1 def would want that as well

**Comment by Stingmeister** [comment]
Would be an awesome feature.

**Comment by castr06** [comment]
please god thatd be amazing if it saved the brightness settings between power cycles

**Comment by joxihan** [comment]
It was added in 1.1 ;)

**Reply by castr06** [reply]
found it thank you ^_^
